modules/apps/object/object-test/src/testFunctional/paths/ObjectNotifications.path

```diff
@@ -45,6 +45,11 @@
 		<td>//*[@class='lexicon-icon lexicon-icon-times-small']</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>VIEW_NOTIFICATION_ENTRIES</td>
+		<td>//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entryLabel}')]/ancestor::div//*[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entryStatus}')]</td>
+		<td></td>
+	</tr>
```

Comment:
Don't we have existing paths for the notification status and the notification entry in the queue?

Reply:
Yeah, we have ! But they don't see two information at the same time, I created this path to see the status and any field on the label together. Do you think that's necessary, or I can do two asserts with two existing paths ?

Reply:
After talking with Jerry, we're going to keep this path and switch our existing tests to use it.